migration/README.md

```diff
@@ -0,0 +1,33 @@
+# Migration
+
+export_database.py contains script to migrate the default database from one instance of MongoDB to another.
+
+## Usage
+
+Script takes two default arguments:
+
+1. URI for source database using flag
+
+    `--source, -s`
+
+2. URI for target database using flag
+
+    `--target, -t`
+
+If the script encounters any problem during the migration with any of the documents, it pushes them to defaulters.bson<br/>
+If a document was already present in the target database, it pushes them to repeated_notices.bson
+
+If you want to further try to push from a bson file, specify it with argument
+<br/>`--try-defaulters or -d`
+
+## Examples
+
+1. Default Export:
+
+    `python export_database.py --source "<Source DB URI>" --target "<Target DB URI>"`
+
+2. Adding documents from bson dump file (generated by script in case of errors) : defaulters.bson
+
+    `python export_database.py --source "<Source DB URI>" --target "<Target DB URI>" --try-defaulters defaulters.bson`
```

Comment:
Have we not updated such that we do not require source when dumping into database from file?